ds_content_types.install

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
please remove this since is not decided yet what to do, it;s an open discussion..

Reply:
Removed.